client/src/buttons.ts

```diff
@@ -1,6 +1,15 @@
 import { ExtensionContext } from 'vscode';
 import * as vscode from 'vscode';
 
+const REACH_COMPILE_S:string = 'Reach Compile';
+const REACH_COMPILE:string = 'reach.compile';
+const REACH_RUN_S:string = 'Reach Run';
```

Comment:
nit: can you separate all the type annotations with spaces? e.g. `const REACH_RUN : string = `